tests/unit/test_storage.py

```diff
@@ -12,10 +12,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import requests
 from cobra import Model
 
 from model import storage
 
 
-def test_get_model():
-    assert type(storage.get('e_coli_core').model) == Model
+def mock_response(self):
+    class Response:
+        status_code = 200
+
+        def json(self):
+            return {
+                'model_serialized': {
+                    'version': "1",
+                    'id': "foo",
+                    'genes': [],
+                    'reactions': [],
+                    'metabolites': [],
+                    'compartments': {},
+                },
+                'organism_id': "bar",
+                'default_biomass_reaction': "baz",
+            }
+
+        def raise_for_status(self):
+            pass
+
+    return Response()
+
+
+def test_get_model(monkeypatch):
```

Comment:
Would a real mocking code be helpful here? https://github.com/pytest-dev/pytest-mock/

Reply:
Looks like a nice lib we should use when requiring the features it provides but I think the monkeypatch serves its purpose well without justifying a new test dependency here.